frontend/src/components/ChangeEmailScreen.js

```diff
@@ -1,75 +1,130 @@
-import React, {useState} from 'react';
-import { Button} from 'react-bootstrap';
+/**
+ * This renders the "change primary email" portion of
+ * The "emails" tab under the "Admin" page.
+ * 
+ */
+
+import React, { useState, useEffect } from 'react';
+import { Button } from 'react-bootstrap';
 import TextField from '@material-ui/core/TextField';
+import Snackbar from '@material-ui/core/Snackbar';
+import { makeStyles } from '@material-ui/core/styles';
 
 import '../css/ChangeEmailScreen.css';
 
 const config = require('../config');
 const BACKEND_URL = config.backend.uri;
 
+const useStyles = makeStyles((theme) => ({
+    form: {
+      //Input Field - Label Layout 
+      '& .MuiFormLabel-root': {
+          color: '#747474',
+        },
+        //Input Field - Border Layout 
+      '& .MuiOutlinedInput-root .MuiOutlinedInput-notchedOutline': {
+          border: '1.5px solid #747474'
+      }
+    }
+}));
+
+export default function AdminMenuItems (props) {
```

Comment:
should be ChangeEmailScreen (this might have been my bad when I first created the file)